src/main/java/com/vonage/quickstart/verify/StartVerification.java

```diff
@@ -19,26 +19,24 @@
  * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
  * THE SOFTWARE.
  */
-package com.nexmo.quickstart.verify;
+package com.vonage.quickstart.verify;
 
-import com.nexmo.client.NexmoClient;
-import com.nexmo.client.verify.VerifyResponse;
-import com.nexmo.client.verify.VerifyStatus;
-
-import static com.nexmo.quickstart.Util.configureLogging;
```

Comment:
This import was dropped which breaks the snippet.

Reply:
fixed